config.js

```diff
@@ -35,10 +37,12 @@ const FHIR = {
   url: process.env.FHIR_URL,
 };
 
-const CHT = {
-  url: process.env.CHT_URL,
-  username: process.env.CHT_USERNAME,
-  password: process.env.CHT_PASSWORD,
+const CHT =()=>{
```

Comment:
lets pass the instance as a param to this function

Reply:
Now passing Instance as a param.